kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkTaskConfig.java

```diff
@@ -93,6 +93,18 @@
   private static final String BIGQUERY_MESSAGE_TIME_PARTITIONING_DOC =
       "Whether or not to use the message time when inserting records. "
       + "Default uses the connector processing time.";
+  
+  public static final String BIGQUERY_PARTITION_DECORATOR_CONFIG =
+          "bigQueryPartitionDecorator";
+  private static final ConfigDef.Type BIGQUERY_PARTITION_DECORATOR_CONFIG_TYPE =
+      ConfigDef.Type.BOOLEAN;
+  //This has been set to true to preserve the existing behaviour. However, we can set it to false if field based partitioning is used in BigQuery
+  public static final Boolean BIGQUERY_PARTITION_DECORATOR_DEFAULT =                 true; 
+  private static final ConfigDef.Importance BIGQUERY_PARTITION_DECORATOR_IMPORTANCE =
+      ConfigDef.Importance.HIGH;
+  private static final String BIGQUERY_PARTITION_DECORATOR_DOC =
+      "Whether or not to append partition decorator to BigQuery table name when inserting records. "
+      + "Default is false. Setting this to true appends partition decorator to table name (e.g. table$yyyyMMdd depending on the configuration set for bigQueryPartitionDecorator)";
```

Comment:
Default is true, right?

Reply:
@mtagle yes, default is `true` to allow the setting of partition based on the value of (existing) `useMessageTimeDatePartitioning` parameter. However, it can be set to `false` if we do not need partition decorator at all.

Reply:
Then the doc should (correctly) say that default is true, rather than that the default is false.

Reply:
@mtagle apologies, missed it earlier. It's fixed now.

Reply:
@mtagle fixed tests as well. Can we merge it now?